docs/release-notes/v0.37.0.md

```diff
@@ -0,0 +1,91 @@
+## Announcing Radius v0.37.0
+
+Today we're happy to announce the release of Radius v0.37.0. Check out the [highlights](#highlights) below, along with the [full changelog](#full-changelog) for more details.
+
+We would like to extend our thanks to all the [new](#new-contributors) and existing contributors who helped make this release possible!
+
+## Intro to Radius
+
+If you're new to Radius, check out our website, [radapp.io](https://radapp.io), for more information. Also visit our [getting started guide](https://docs.radapp.io/getting-started/) to learn how to install Radius and create your first app.
+
+## Highlights
+
+### Radius is merged with the official Bicep
+Previously Radius used a fork of the Bicep to author and deploy Radius and AWS types. With 0.37 release we have merged our extensibility updates with the upstream and are deprecating the temporary fork of Bicep. To use Radius features with the official Bicep compiler, you need to create a [`bicepconfig.json`]((https://docs.radapp.io/guides/tooling/bicepconfig/overview/)) with the extensions to author and deploy Radius and AWS types. Please checkout the breaking changes section for detailed guidance.
+
+### Radius AWS Cloud provider supports IAM Roles for Service Accounts (IRSA)
+You can now configure your AWS cloud provider to use AWS federated identity (IRSA) to deploy and manage AWS resources. With this support, infrastructure operators are not burdened with the rotation of the credentials anymore. More information on [how to configure the AWS IRSA](https://docs.radapp.io/guides/operations/providers/aws-provider/howto-aws-provider-irsa/)
+
+### Test Improvements
+We separated the cloud tests and non-cloud tests in our functional  test workflows to improve the test verification process for contributors. In the new workflow the non-cloud test workflows will trigger automatically when pull requests are submitted, while tests using cloud resources will need approval from approvers/maintainers to be triggered. We also fixed an issue with `magpiego` image failing for MacOS+M1 chip users by publishing a multi-architectural image.
+
+## Breaking changes
+
+**Radius merge to official Bicep** introduces some breaking changes. To use Radius features with the official Bicep compiler please do the following:
```

Comment:
Formatting looks a little off. I think newlines need to be added


![Screenshot 2024-08-19 at 9 58 59 AM](https://github.com/user-attachments/assets/baaa3be8-b315-495f-a5a5-c27751a5cf7c)

Reply:
fixed it